app/src/main/java/org/sopt/carrot/presentation/productDetailScreen/ProductDetailScreen.kt

```diff
@@ -0,0 +1,143 @@
+
+import androidx.compose.animation.animateColorAsState
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.zIndex
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.navigation.NavHostController
+import org.sopt.carrot.presentation.ScreenRoutes
+import org.sopt.carrot.presentation.productDetailScreen.components.KeywordAlertSection
+import org.sopt.carrot.presentation.productDetailScreen.components.ProductBottomBar
+import org.sopt.carrot.presentation.productDetailScreen.components.ProductImageSection
+import org.sopt.carrot.presentation.productDetailScreen.components.ProductInfoSection
+import org.sopt.carrot.presentation.productDetailScreen.components.ProductTopBar
+import org.sopt.carrot.presentation.productDetailScreen.components.RecommendProductSection
+import org.sopt.carrot.presentation.productDetailScreen.components.RelatedProductSection
+import org.sopt.carrot.presentation.productDetailScreen.components.UserInfoSection
+import org.sopt.carrot.presentation.productDetailScreen.model.ProductDetailUiState
+
+@Composable
+fun ProductDetailScreen(
+    navController: NavHostController,
+    modifier: Modifier = Modifier
+) {
+    val onBackClick = { navController.navigate(ScreenRoutes.TITLE_SEARCH) }
+    val onHomeClick = { navController.navigate((ScreenRoutes.EXAMPLE_SCREEN_1)) }
+    val viewModel: ProductDetailViewModel = remember { ProductDetailViewModel() }
+    val uiState: ProductDetailUiState by viewModel.uiState.collectAsStateWithLifecycle()
+
+    val scrollState = rememberLazyListState()
+    val isScrolledPastImage =
+        scrollState.firstVisibleItemIndex > 0 || scrollState.firstVisibleItemScrollOffset > 0
+
+    val backgroundColor by animateColorAsState(
+        targetValue = if (isScrolledPastImage) Color.White else Color.Transparent,
+        animationSpec = tween(durationMillis = 500),
+        label = ""
+    )
+
+    Scaffold(
+        topBar = {
+            ProductTopBar(
+                onBackClick = onBackClick,
+                onHomeClick = onHomeClick,
+                backgroundColor = backgroundColor,
+                modifier = Modifier.zIndex(1f).background(backgroundColor)
```

Comment:
p3: 줄바꿈 한번 부탁드립니다!👨🏻‍🌾
```suggestion
                modifier = Modifier
                            .zIndex(1f)
                            .background(backgroundColor)
```

Reply:
적용했습니다!!